docs/how_to_guides/how_to_scale_chemical_reactions.rst

```diff
@@ -0,0 +1,158 @@
+    is named **thermo_params**, your **GenericReactionParameterBlock** is named
+    **rxn_params**, your unit model is named **unit**, and your **FlowsheetBlock**
+    is named **fs**. For information on setup of the basic chemistry modules,
+    see :ref:`How to setup simple chemistry<how_to_setup_simple_chemistry>`.
+
+
+.. note::
+
+    The scaling factor constants used in these demonstrations are the best found
+    for these types of problems based on numerical testing of a variety of problems.
+    These values can and may need to change for your particular problem.
+
+
+.. note::
+
+    IMPORTANT: Scaling of just the chemical reactions is insufficient for solving
+    a chemistry module. User's MUST also scale the chemical species in the system
+    due to the dilute nature of aqueous chemistry. See
+    :ref:`How to scale chemical species<how_to_scale_chemical_species>`.
+
+
+Types of Reactions
+------------------
+
+1. Inherent Reactions using log_power_law_equil form
+2. Equilibrium Reactions using log_power_law_equil form
+3. Rate Reactions using power_law_rate form
+4. Stoichiometric Reactions (Documentation Pending)
+
+
+Inherent Reactions using log_power_law_equil form
+-------------------------------------------------
+
+Inherent reactions (see :ref:`How to use inherent reactions<how_to_use_inherent_reactions>`)
+are placed into the **GenericParameterBlock**
+rather than the **GenericReactionParameterBlock**. As such, the scaling methods employed
+must be applied to the **thermo_params** for this set of reactions. The code segment below
+shows a demonstration of scaling factors that generally work well for these types of reactions.
+
+
+Inherent Scaling Demonstration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block::
+
+    # Iterate through the reactions to set appropriate eps values
+    factor = 1e-4
+    for rid in model.fs.thermo_params.inherent_reaction_idx:
+        scale = value(model.fs.unit.control_volume.properties_out[0.0].k_eq[rid].expr)
+        # Want to set eps in some fashion similar to this
+        if scale < 1e-16:
+            model.fs.thermo_params.component("reaction_"+rid).eps.value = scale*factor
+        else:
+            model.fs.thermo_params.component("reaction_"+rid).eps.value = 1e-16*factor
+
+    for i in model.fs.unit.control_volume.inherent_reaction_extent_index:
+        scale = value(model.fs.unit.control_volume.properties_out[0.0].k_eq[i[1]].expr)
+        iscale.set_scaling_factor(model.fs.unit.control_volume.inherent_reaction_extent[0.0,i[1]], 10/scale)
+        iscale.constraint_scaling_transform(model.fs.unit.control_volume.properties_out[0.0].
+                inherent_equilibrium_constraint[i[1]], 0.1)
+
+
+What this code segment is doing is to first assign an **eps** value for the **log_power_law_equil**
+function. The default value is generally insufficient, so we change that value to always be at least
+4 orders of magnitude less than the **k_eq** value calculated for each reaction.
```

Comment:
Since you apply 10/scale in line 81, should it technically be 5 orders of magnitude?

Reply:
No. The eps value does not impact the scaling factors (significantly) because of how it is used in the function. It is a smoothing parameter for the log function. However, it is important and convenient to set this value here. 

Also, there would not be much need to go into details on the eps value, because it will go away with a future iteration of IDAES.